components/commons/overlay-coretokens.css

```diff
@@ -0,0 +1,54 @@
+/*
+Copyright 2019 Adobe. All rights reserved.
```

Comment:
```suggestion
Copyright 2022 Adobe. All rights reserved.
```

Reply:
Waiting to hear from @GarthDB about this, but I don't believe additional work is required on overlay for popover.